knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json

```diff
@@ -25,7 +25,7 @@
       "routing": {
         "GET": [
           {
-            "path": "/content/.*",
+            "path": ".*\\.html",
```

Comment:
The way I understand it, the previous config would accept query strings and the new one doesn't. Is my reasoning correct? If so, is this the desired behaviour?